.github/workflows/ci.yml

```diff
@@ -39,38 +39,38 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: Linux (GCC 7, 32-bit)
-            os: ubuntu-20.04
-            CC: gcc-7
-            CXX: g++-7
+          - name: Linux (GCC 9, 32-bit)
+            os: ubuntu-22.04
+            CC: gcc-9
+            CXX: g++-9
```

Comment:
Had to change to `gcc-9` because `multilib` was not available otherwise.

Reply:
Yeah, as discussed on discord, `22.04` has nothing below `gcc-10` on `jammy` and `gcc-9` on `jammy-updates`.